docs/how-tos/protocol.rst

```diff
@@ -2,3 +2,541 @@
+            
+            # Handle extension from previous results if needed
+            if extends is not None:
+                # Extract useful information from the previous run
+                # This might be final coordinates, equilibrated structures, etc.
+                starting_point = extends.protocol_unit_results[-1].outputs
+            else:
+                starting_point = None
+            
+            # Create the setup unit (runs once)
+            setup = SetupUnit(
+                name="system_setup",
+                stateA=stateA,
+                stateB=stateB,
+                mapping=mapping,
+                settings=self.settings,
+                starting_point=starting_point
+            )
+            
+            # Create multiple independent simulation units
+            simulations = []
+            for i in range(self.settings.n_repeats):
+                sim_unit = SimulationUnit(
+                    name=f"simulation_{i}",
+                    setup_result=setup,  # This creates the dependency
+                    window=i,
+                    settings=self.settings
+                )
+                simulations.append(sim_unit)
+            
+            # Create analysis unit that depends on all simulations
+            analysis = AnalysisUnit(
+                name="final_analysis",
+                simulation_results=simulations,  # Depends on all simulations
+                settings=self.settings
+            )
+            
+            # Return all units - dependencies are implicit from constructor args
+            return [setup, *simulations, analysis]
+        
+        def _gather(self, protocol_dag_results: Iterable[ProtocolDAGResult]) -> dict[str, Any]:
+            """Aggregate results from multiple ProtocolDAG executions."""
+            # This method combines results from multiple independent protocol runs
+            # into data that the ProtocolResult can use to compute estimates
+            
+            free_energies = []
+            all_logs = []
+            
+            for dag_result in protocol_dag_results:
+                # Find the terminal (final) unit results
+                for unit_result in dag_result.terminal_protocol_unit_results:
+                    if unit_result.name == "final_analysis":
+                        free_energies.append(
+                            unit_result.outputs["total_free_energy"]
+                        )
+                        all_logs.extend(unit_result.outputs["simulation_logs"])
+            
+            return {
+                "free_energies": free_energies,
+                "logs": all_logs
+            }
+
+
+Step 5: Add validation (optional)
+----------------------------------
+
+You can add custom validation to check that inputs are compatible with your protocol:
+
+.. code-block:: python
+
+    class MyProtocol(Protocol):
+        # ... other methods ...
+        
+        def _validate(
```

Comment:
Also point to the default validate method as it might not be appropriate for this protocol or they might impliment redundant validation.